moskito-core/src/main/java/net/anotheria/moskito/core/producers/AbstractStats.java

```diff
@@ -65,7 +65,7 @@
 	/**
 	 * Contains all stat values that are part of this StatsObject. This is used to properly destroy them.
 	 */
-	private LinkedList<StatValue> statValuesList = new LinkedList<StatValue>();
+	private Collection<StatValue> statValuesList = new LinkedList<>();
```

Comment:
If you don't use some parts of a concrete implementation, is a recommended way to use the more abstract interface or class. In genereal this leads to a better encapsulation, because the implementation can be easily changed. If the HashMap type could lead to better performance here (or sometimes you need another behaviour like element ordering), you are able to use the type without changing the remaining code.